src/commands/plugins/trust/verify.ts

```diff
@@ -57,21 +59,22 @@ export class Verify extends SfdxCommand {
       cliRoot: get(this.config, 'root') as string,
     };
 
-    this.logger.debug(`cacheDir: ${configContext.cacheDir}`);
-    this.logger.debug(`configDir: ${configContext.configDir}`);
-    this.logger.debug(`dataDir: ${configContext.dataDir}`);
+    logger.debug(`cacheDir: ${configContext.cacheDir}`);
+    logger.debug(`configDir: ${configContext.configDir}`);
+    logger.debug(`dataDir: ${configContext.dataDir}`);
 
     vConfig.verifier = Verify.getVerifier(npmName, configContext);
 
-    vConfig.log = this.ux.log.bind(this.ux) as (msg: string) => void;
+    // TODO: how to pass the new logger to it
+    // vConfig.log = this.ux.log.bind(this.ux) as (msg: string) => void;
```

Comment:
This can be removed, the class already uses cli-ux:
https://github.com/salesforcecli/plugin-trust/blob/sm/sf-ify-plus-nuts-fix/src/shared/installationVerification.ts#L455